### DISCUSSION TITLE
point_0727

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#338] [Roy]
"20190913T022551_20190913T022548_T51RUQ"
![image](https://user-images.githubusercontent.com/54947195/147799541-4dac966e-70f3-49fb-bfc9-18ee48e2dfde.png)

"20190610T022559_20190610T023806_T51RUQ"
![image](https://user-images.githubusercontent.com/54947195/147799572-9b605003-2ea0-4a19-8990-d960fc7d3f03.png)

"20190504T023559_20190504T024226_T51RUQ"
![image](https://user-images.githubusercontent.com/54947195/147799617-4547d552-0103-490d-bcc0-6e033259ecc3.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190610T022559_20190610T023806_T51RUQ"
![Captura](https://user-images.githubusercontent.com/86502862/153695143-33fef38e-de92-402f-a97b-9b9a64d9e233.JPG)
